generate.php

```diff
@@ -8,10 +8,11 @@
 // --------------------------------------------------------------- //
 // Parse the repositories.json file to extract extension locations //
 // --------------------------------------------------------------- //
-$repositories = json_decode(file_get_contents('repositories.json'), true);
-if (JSON_ERROR_NONE !== json_last_error()) {
+try {
+	$repositories = json_decode(file_get_contents('repositories.json'), true, 512, JSON_THROW_ON_ERROR);
+} catch (Exception $exception) {
 	echo 'The repositories.json file is not a valid JSON file.', PHP_EOL;
-	die;
+	exit(1);
```

Comment:
No need to change it, but I often use `die(1)` (or another code) for errors and `exit(0)` when there is no error - the two are synonyms